### PR TITLE
added wesley testimonial

### PR DESCRIPTION
**Pull request by DJOcean**

### CODE REVIEW

**Comment by DJOcean** [comment]
Also, a headshot of Wesley would be nice. And maybe a last name or initial

File: styles/main.css [hunk 1]
```diff
@@ -280,9 +280,24 @@ header {
     font-weight: 400;
     margin-bottom: 8px; }
   .testimonial-section p {
+    max-height: 200px;
     font-size: 14px;
     text-align: justify;
     font-weight: 100; }
+  .testimonial-section span {
+    display: none; }
+  .testimonial-section .wesley-quote:hover span {
```
Comment (theWickedWebDev) [inline styles/main.css]: why does wesley have his own class?

File: index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <base href="http://thewickedwebdev.github.io/blvenyrbdy/">
+    <!--<base href="http://thewickedwebdev.github.io/blvenyrbdy/">-->
```
Comment (theWickedWebDev) [inline index.html]: why did you comment this out?

File: index.html [hunk 3]
```diff
@@ -55,8 +55,8 @@ <h2 class="animated pulse">Helping you get <strong>strong</strong> through the p
...
...of that doesn't always provide the best motivation for being healthy and fit. So when I met my amazing girlfriend who has been taking Robin's classes for years now, I was a little motivated to do something more. However if you're anything like me, you don't want to go to the gym alone or feel like you are being judged. So what did I do? I finally tried my girlfriend's class. I was shy, awkward, and not sure I'd ever go again because let's face it, exercise is painful. Yet when I finished, I felt great! A feeling that had long since been lost in the fitness category! So I went again, and again, and I will never quit now! When I am there I don't feel judged, I don't feel watched. I feel alive! I feel encouraged! I believe! Robin always says, breath and believe. Breathing is something I am forced to do, but believing is something I choose to do because Robin, my girlfriend, and the others in the class have helped me to rediscover that deep feeling we lose so easily. I have so much faith in myself just because my girlfriend and I go to class every week. I encourage anyone to join the class, it's a great exercise, both physically and mentally, and it's a great date night too!</span></p>
```
Comment (theWickedWebDev) [inline index.html]: why are you making another class for the only p tag in that div? you already have another class you can use

File: styles/main.css (hunk 1 above)
Comment (theWickedWebDev) [inline styles/main.css]: why cant you use:
`.testimonial-section .tp01 p:hover span {`

**Comment by DJOcean** [comment]
Okay i left the class but changed the name so that it would effect each of them the same. Also added Becky's testimonial. I had that one line of code commented out because when i was working on the calandar it wouldnt work and i forgot to uncoimment it out, so i put it back. I also made the third testimonial react the same as the others, it just mimicks the 2nd until we get a new one.

**Comment by DJOcean** [comment]
ignore that "added test" thing. that was me messing up and trying to figure it out

**Comment by theWickedWebDev** [comment]
LGTM

**Pull request merged**